frontend/app/routes/_index/components/top-navigation/top-navigation.module.css

```
@@ -0,0 +1,31 @@
+    var(--color-success) 65%,
+    var(--color-info) 85%,
+    var(--color-primary) 100%
+  );
+  background-size: 300% 100%;
+  background-position: 0% 50%;
+  background-clip: padding-box;
+  animation: updateAvailableShift 6s ease-in-out infinite;
+  box-shadow: none;
+}
+
+@keyframes updateAvailableShift {
```

Stylelint (17.14.0)

[error] Expected keyframe name "updateAvailableShift" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)